### PR TITLE
Add Windows getting started setup instructions

### PR DESCRIPTION
**Pull request by Nicell**
These instructions were created by using the Windows instructions listed here: https://docs.zephyrproject.org/latest/getting_started/index.html

I haven't tested this fully as I don't have a Windows VM, but it should work properly from when I remember when setting up my Windows environment.

### CODE REVIEW

File: docs/docs/dev-setup.md [hunk 1]
```diff
@@ -166,15 +181,23 @@ We suggest two main [options](https://docs.zephyrproject.org/latest/guides/env_v
 
 To load the Zephyr environment properly for just one transient shell, run the following from your ZMK checkout directory:
 
+Unix:
```
Comment (Nicell) [inline docs/docs/dev-setup.md:184]: Not sure if this is the best way to split up Windos/Unix. Would be nice to have tabs to click on like in the Zephyr docs.

File: docs/docs/dev-setup.md [hunk 2]
```diff
@@ -166,15 +181,23 @@ We suggest two main [options](https://docs.zephyrproject.org/latest/guides/env_v
...
+```
+source zephyr/zephyr-env.cmd
+```
+
 #### All Shells
 
 To load the environment variables for your shell every time,
 append the existing `~/.zephyrrc` file to your shell's RC file and then start a new shell.
 
+*Skip on Windows*
```
Comment (Nicell) [inline docs/docs/dev-setup.md:199]: Not sure where else to put this.

**Comment by petejohanson** [comment]
@Nicell I've incorporated the one build fix already into `main`, and have opened #55 to address the "multiple OS" issues w/ the docs. Care to give that PR a review? TIA.

**Comment by petejohanson** [comment]
Closing this, since we merged #55

**Pull request closed without merging**